ovs/extensions/plugins/apiclient.py

```diff
@@ -86,8 +104,9 @@ def _call(self, method, url, data=None, timeout=None, clean=False):
                   'headers': self._base_headers,
                   'verify': False,
                   'timeout': timeout}
-        if data is not None:
-            kwargs['data'] = data
+        for key, val in [('json', json), ('data', data)]:
```

Comment:
Could you provide some comments on why you do this?

Reply:
Requests library can both take in 'data' or 'json' keyword.
When data is given: no extra heading is added and the data is serialized as json
When json is given: the 'Content type: Application/json' header is added.
The loop is to provide both options